documentation/extracts.md

```diff
@@ -85,9 +90,22 @@ Some user groups have specific custom exports [which execute via the `CustomExpo
 
 The custom exports are based on materialized views, [defined in `hocs-audit`](https://github.com/UKHomeOffice/hocs-audit/blob/main/config/materializedviews/Audit-Schema-DataUpdates.sql).
 
-The materialized views can be generated on a scheduled job processing the report ahead of time. This means reports can be ready for download immediately, but with the drawback they do not present a real-time snapshot.
+#### Scheduled jobs
```

Comment:
Might be worth a one-liner to indicate that these directly interact with the DB and not the underlying service